src/datasets/features.py

```diff
@@ -696,7 +738,13 @@ class Sequence:
 
 
 def get_nested_type(schema: FeatureType) -> pa.DataType:
-    """ Convert our Feature nested object in an Apache Arrow type """
+    """
```

Comment:
docstring I left while I was working locally - happy to remove or keep based on guidance from the maintainers!